src/data/ordmap/ordset.lean

```diff
@@ -543,9 +543,10 @@ theorem dual_insert [preorder α] [is_total α (≤)] [@decidable_rel α (≤)]
   ∀ t : ordnode α, dual (ordnode.insert x t) = @ordnode.insert αᵒᵈ _ _ x (dual t)
 | nil := rfl
 | (node _ l y r) := begin
-  rw [ordnode.insert, dual, ordnode.insert, order_dual.cmp_le_flip, ← cmp_le_swap x y],
+  have : @cmp_le αᵒᵈ _ _ x y = cmp_le y x := rfl,
+  rw [ordnode.insert, dual, ordnode.insert, this, ← cmp_le_swap x y],
```

Comment:
IMHO, we should use `order_dual` in definitions and lemmas about `ordnode.dual` but I don't want to do this refactor myself (also, I don't want to to any refactor there without Mario's approval).